### ISSUE TITLE
add to MRU history only after releasing ctrl (native)

### ISSUE THREAD

**Issue by dwelle** [oggy/tab-switcher#45]
If I want to e.g. go to 5th tab by using native's ctrl-tab so that it doesn't invoke the tab-list (which is distracting to me), it's not possible if `reorder` option is on, since the tab is added to MRU history after first `ctrl-tab`, thus reordering it immediately afterwards.

The behavior then is

- infinitely cycling between the two (if Atom MRU is disabled)
- MRU cycling (if Atom MRU enabled) -- which again isn't what I want.

In other words, what I want is to be able to use Atom native ctrl-tab to linearly move to nth-tab, without invoking the tab-switcher's tab-list.

End goal being using the native tab-bar as a tab-list when switching, instead of using the tab-switcher's -- best of both worlds IMO.

**Comment by oggy** [comment]
If you find the tab-list distracting, then I'm not sure you actually want tab-switcher.

Would it be enough to just hook into the command that runs when you release ctrl to move the tab to the left?

```
atom.commands.add 'atom-workspace',
  'pane:move-active-item-to-top-of-stack': ->
    pane = atom.workspace.getActivePane()
    item = pane.getActiveItem()
    pane.moveItem(item, 0)
```
